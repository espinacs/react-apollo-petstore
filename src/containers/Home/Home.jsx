import React from 'react'

const DaHoma = () => <h1>hello handsome</h1>

export default DaHoma