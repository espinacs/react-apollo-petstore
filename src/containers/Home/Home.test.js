import React from 'react';
import { render, cleanup } from 'react-testing-library';
import Home from '.';

afterEach(cleanup);

describe('Home', () => {
  test('renders default', () => {

    const { container } = render(<Home />)
    expect(container.firstChild).toMatchSnapshot();
  });
});