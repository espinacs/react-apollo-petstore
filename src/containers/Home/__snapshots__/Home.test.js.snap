// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders default 1`] = `
<h1>
  hello handsome
</h1>
`;
